schema/ruleset_schema.json

```diff
@@ -115,9 +124,40 @@
                     "required": [
                         "name"
                     ],
+                    "additionalProperties": true
+                }
+            },
+            "required": [
+                  "run_playbook"
+            ]
+        },
+        "run-module-action": {
+            "type": "object",
+            "properties": {
+                "run_module": {
+                    "type": "object",
+                    "properties": {
+                        "copy_files": { "type": "boolean" },
+                        "name": {"type": "string" },
+                        "post_events": { "type": "boolean" },
+                        "set_facts": { "type": "boolean" },
+                        "verbosity": {"type": "integer" },
+                        "var_root": {"type": ["string","object"] },
+                        "json_mode": { "type": "boolean" },
+                        "retry": { "type": "boolean" },
+                        "retries": { "type": "integer" },
+                        "delay": { "type": "integer" },
```

Comment:
delay type should be  `number`, not limit to integer. Same for run-playbook-action.